### PR TITLE
Implemented compass position for Android

### PR DESCRIPTION
**Pull request by mfazekas**
Fixes: #757

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -29,10 +29,53 @@ A clear and concise description of what the bug is.
...
+Example:
+
+```sh
+react-native init sample --version react-native@0.60.5
+cd sample
+npm install react-native-mapbox-gl/maps#master --save
+# or released version `npm install @react-native-mapbox-gl/maps@8.0.0-rc1 --save`
+react-native run-android
+```
+
```
Comment (ferdicus) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Was this supposed to be in this PR?

File: example/src/examples/BugReportExample.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        <Camera centerCoordinate={[-74.00597, 40.71427]} zoomLevel={14} />
+        <ShapeSource id="idStreetLayer" shape={aLine}>
+          <LineLayer id="idStreetLayer" />
+        </ShapeSource>
+      </MapView>
+    );
+  }
+}
+
+export default BugReportExample;
```
Comment (ferdicus) [inline example/src/examples/BugReportExample.js]: I think this is related to another PR maybe?

File: docs/MapView.md [hunk 3]
```diff
@@ -18,7 +18,7 @@
 | attributionPosition | `union` | `none` | `false` | Adds attribution offset, e.g. `{top: 8, left: 8}` will put attribution button in top-left corner of the map |
 | logoEnabled | `bool` | `true` | `false` | Enable/Disable the logo on the map. |
 | compassEnabled | `bool` | `none` | `false` | Enable/Disable the compass from appearing on the map |
-| compassViewPosition | `number` | `none` | `false` | Change corner of map the compass starts at. (iOS only) 0: TopLeft, 1: TopRight, 2: BottomLeft, 3: BottomRight |
+| compassViewPosition | `number` | `none` | `false` | Change corner of map the compass starts at. 0: TopLeft, 1: TopRight, 2: BottomLeft, 3: BottomRight |
```
Comment (ferdicus) [inline docs/MapView.md]: the api for this feels a bit clunky - why not expose it like this:
https://github.com/react-native-mapbox-gl/maps/blob/45faf91c9543dc4f1a64489f443dd1322495d118/javascript/components/Camera.js#L565-L569

Are you thinking about compatibility for people already using this on iOS?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (mfazekas) [reply .github/ISSUE_TEMPLATE/bug_report.md]: No. And yes because i'm lazy.

File: example/src/examples/BugReportExample.js (hunk 2 above)
Comment (mfazekas) [reply example/src/examples/BugReportExample.js]: Sure i'll split this to 2 PR.

File: docs/MapView.md (hunk 3 above)
Comment (mfazekas) [reply docs/MapView.md]: Yep, i wanted to add android parity. We can and should introduce symbols, not just magic constants.

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (ferdicus) [reply .github/ISSUE_TEMPLATE/bug_report.md]: aren't we all :D

**Pull request merged**